package.json

```diff
@@ -21,6 +21,9 @@
     "build": "tsc --noEmit --project .",
     "buildDocs": "typedoc"
   },
+  "dependencies": {
+    "json-rpc-engine": "^6.1.0"
```

Comment:
Hmm, I was hoping to keep this package as _just_ types. Could the JSON RPC engine types be incorporated into this package as well? We could put all of the RPC-related types in a common directory.

Reply:
We can 👍

Reply:
cc @rekmarks , thoughts on whether this is a good idea?

I was anticipating that we'd want to avoid circular dependencies between types if possible (sounds like bad news for anything consuming those types, like linters, tsc, IDEs, typedoc, etc.), and that the types in this repo might get used in `json-rpc-engine`.

But looking at the full set of types brought in here... it's substantial. I am unsure.

Reply:
I discussed this with @Gudahtt, and I think we can merge this. While the implementation of the `json-rpc-engine` types are closely coupled to the implementation thereof, they are widely used and ought to exhibit very low levels of churn.